scripts/feedme.coffee

```diff
@@ -0,0 +1,120 @@
+# Description:
+#   Shows what"s edible.
+#
+# Notes:
+#   This includes
+#    * fetching from fabrik
+#    * fetching from ernis
+#
+# Author: cholter
+
+urllib = require "urllib-sync"
+jsdom = require "jsdom"
+require "datejs"
+
+DAY = 1000 * 60 * 60  * 24
+
+getDateDiff = (date1, date2, interval) ->
+  return Math.round((date1.getTime() - date2.getTime()) / interval)
+
+fetch = (target) ->
+  return urllib.request(target).data
+
+
+class Fabrik
+  target : "http://www.diefabrik.co.at/mittagsmenue/index.html"
+  holidayMagic : "urlaub"
+  errors:
+	  closed : "fabrik is closed"
+	  menuFromPast :"the menu is outdated"
+	  menuFromFuture :"the menu is from the future"
+	  resting :"fabrik is on a day off"
+	  holiday :"fabrik is probably on holiday, fall back to manual check"
+
+  constructor: (@robot) ->
+
+  checkHoliday: (text) =>
+    return true if text.toLowerCase().indexOf(@holidayMagic) > 0
```

Comment:
Assuming `return true if x` returns `false` if `not x`, you can omit this altogether and write `return x`.